core/templates/components/feedback-shared/feedback-detail-page/feedback-detail-page.component.css

```
@@ -1,35 +1,263 @@
+  padding: 4px 0;
+}
+
+.oppia-feedback-detail-field-label {
+  color: black;
+  flex: 0 0 110px;
+  font-weight: 500;
+}
+
+.oppia-feedback-detail-field-value {
+  color: #262626;
+  word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)